jerry-core/debugger/debugger-ws.c

```diff
@@ -65,24 +60,20 @@ typedef struct
  * Close the socket connection to the client.
  */
 static void
-jerry_debugger_close_connection_tcp (bool log_error) /**< log error */
+jerry_debugger_close_connection_transport (void)
```

Comment:
Is there a plan to log connection error in the future?

Reply:
yeah, we can just change to all the handlers Apis to take the same log_error parameter

Reply:
It looks like only this function has such an option.